examples/rest2/trade-history.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+process.env.DEBUG = 'bfx:examples:*'
+
+const args = process.argv
+const debug = require('debug')('bfx:examples:rest2-trade-history')
+
+if (args.length < 3) {
+  debug('error: symbol required (i.e: npm run trade-history ETHUSD')
+  process.exit(1)
+}
+
+const Table = require('cli-table2')
+const bfx = require('../bfx')
+const rest = bfx.rest(2, { transform: true })
+const pair = String(args[2])
+const symbol = pair[0] === 't' ? pair : `t${pair}`
+
+// TODO: natural lang start/end query args (a library must exist)
+const START = Date.now() - (30 * 24 * 60 * 60 * 1000 * 1000)
+const END = Date.now()
+const LIMIT = 25
+
+const table = new Table({
+  colWidths: [12, 13, 20, 20, 14, 14, 14, 20],
+  head: [
+    'Trade ID', 'Order ID', 'Created', 'Type', 'Exec Amount', 'Exec Price',
+    'Price', 'Fee'
```

Comment:
some fields are empty for me...

![screen shot 2018-04-27 at 12 12 15](https://user-images.githubusercontent.com/298166/39357843-54bf15a2-4a14-11e8-823e-115efb76a4d0.png)

Reply:
It needs https://github.com/bitfinexcom/bitfinex-api-node/pull/298

Reply:
This is an API data issue, if the fields themselves are null in the ws2 packet they will show up empty.